src/Traits/Hooks/UseDashboardZoneOne.php

```diff
@@ -143,6 +149,8 @@ private function ensurePsAccountIsEnabled(): bool {
 
     private function ensurePsEventbusEnabled()
     {
+        if (version_compare(_PS_VERSION_, "9.0.0", ">=")) return false;
```

Comment:
Don't forget to remove this after publication of v9.0.0 on addons and/or official releases for the module